Bump gettext macro numbers as required by newer gettext built by OE.

Upstream-Status: Pending

Signed-off-by: Joshua Lock <josh@linux.intel.com>

Index: texinfo-4.13/configure.ac
===================================================================
--- texinfo-4.13.orig/configure.ac
+++ texinfo-4.13/configure.ac
@@ -193,7 +193,7 @@ AM_CONDITIONAL(INSTALL_WARNINGS,
 
 # i18n support.  To update to a new version of gettext, run:
 # gettextize -f -c
-AM_GNU_GETTEXT_VERSION([0.17])
+AM_GNU_GETTEXT_VERSION([0.18])
 AM_GNU_GETTEXT([external])
 
 # The test suite.
Index: texinfo-4.13/po/Makefile.in.in
===================================================================
--- texinfo-4.13.orig/po/Makefile.in.in
+++ texinfo-4.13/po/Makefile.in.in
@@ -8,8 +8,8 @@
 # Please note that the actual code of GNU gettext is covered by the GNU
 # General Public License and is *not* in the public domain.
 #
-# Origin: gettext-0.17
-GETTEXT_MACRO_VERSION = 0.17
+# Origin: gettext-0.18
+GETTEXT_MACRO_VERSION = 0.18
 
 PACKAGE = @PACKAGE@
 VERSION = @VERSION@
